src/affine.jl

```diff
@@ -1,7 +1,7 @@
 function linmap(mat, img::AbstractArray{T,N}, SD=Matrix(1.0*I,N,N), initial_tfm=IdentityTransformation()) where {T,N}
```

Comment:
I'd get rid of `SD` as an argument here and use it just before you apply the result to the array in `qd_affine_coarse` and `qd_affine_fine`. Otherwise you're inconsistent between how the rigid code works and how this works. The new rule is "apply SD immediately before you warp the array." And never any time else.